src/styles/getTooltipStyles.js

```diff
@@ -0,0 +1,117 @@
+import {Dimensions} from 'react-native';
+import spacing from './utilities/spacing';
+import styles from './styles';
+import themeColors from './themes/default';
+import fontFamily from './fontFamily';
+import variables from './variables';
+
+const GUTTER_WIDTH = 16;
+
+/**
+ * The Expensify.cash repo is very consistent about doing spacing in multiples of 4.
+ * In an effort to maintain that consistency, we'll make sure that any distance we're shifting the tooltip
+ * is a multiple of 4.
+ *
+ * @param {Number} n
+ * @returns {Number}
+ */
+function roundToNearestMultipleOfFour(n) {
+    if (n > 0) {
+        return Math.ceil(n / 4.0) * 4;
+    }
+
+    if (n < 0) {
+        return Math.floor(n / 4.0) * 4;
+    }
+
+    return 0;
+}
+
+export default function getTooltipStyles(
+    interpolatedSize,
+    xOffset,
+    yOffset,
+    width,
+    height,
+    tooltipWidth,
+    tooltipHeight,
+) {
+    // Determine if the tooltip should display below the wrapped component.
+    const windowWidth = Dimensions.get('window').width;
```

Comment:
We maybe should pass this into this method as an optimization. We should only need to know if the window width has updated when it changes. Recently added an HOC we can use for this here.

https://github.com/Expensify/Expensify.cash/blob/master/src/components/withWindowDimensions.js